### PR TITLE
Rebranding to switchio

### PR DESCRIPTION
**Pull request by tgoodlet**
A massive rename for this hard fork.

@vodik ping ping!

### CODE REVIEW

File: conf/ci-minimal/dialplan/switchydp.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- A context for relinquishing control of all calls to Switchy, the inbound ESL client -->
```
Comment (vodik) [inline conf/ci-minimal/dialplan/switchydp.xml]: Missed one?

File: conf/switchydp.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- A context for relinquishing control of all calls to Switchy, the inbound ESL client -->
```
Comment (vodik) [inline conf/switchydp.xml]: And here.

File: docs/apps.rst [hunk 3]
```diff
@@ -102,14 +102,14 @@ Examples
 
 TonePlay
 ********
-As a first example here is the :py:class:`~switchy.apps.players.TonePlay`
+As a first example here is the :py:class:`~switchio.apps.players.TonePlay`
 app which is provided as a built-in for Switchy
```
Comment (vodik) [inline docs/apps.rst]: And here

File: docs/callgen.rst [hunk 4]
```diff
@@ -24,9 +24,9 @@ a sequence of slave server host names::
 .. note::
     If using ESL ports different then the default *8021*, simply pass
     a sequence of `(host, port)` socket pairs to the
-    :py:class:`~switchy.apps.call_gen.get_originator` factory.
+    :py:class:`~switchio.apps.call_gen.get_originator` factory.
 
-Now we have a binding to an :py:class:`~switchy.apps.call_gen.Originator`
+Now we have a binding to an :py:class:`~switchio.apps.call_gen.Originator`
 instance which is a non-blocking Switchy :doc:`application <apps>` allowing us
```
Comment (vodik) [inline docs/callgen.rst]: And here

File: docs/callgen.rst [hunk 5]
```diff
@@ -81,45 +81,45 @@ the outbound session's post-connect execution::
         proxy='doggy@intermediary_hostname:5060',
     )
     >>> originator.originate_cmd  # show the rendered command str
-    ['originate {{originator_codec=PCMU,switchy_client={app_id},
+    ['originate {{originator_codec=PCMU,switchio_client={app_id},
     originate_caller_id_name=Mr_Switchy,originate_timeout=60,absolute_codec_string=,
```
Comment (vodik) [inline docs/callgen.rst]: And here

File: docs/callgen.rst [hunk 6]
```diff
@@ -188,14 +188,14 @@ Slave cluster
 *************
 In order to deploy call generation clusters some slightly more advanced
 configuration steps are required to properly provision the
-:py:class:`switchy.apps.call_gen.Originator`. As mentioned previous,
+:py:class:`switchio.apps.call_gen.Originator`. As mentioned previous,
 this involves use of handy cluster helper components provided with
 Switchy.
```
Comment (vodik) [inline docs/callgen.rst]: And here

File: docs/quickstart.rst [hunk 7]
```diff
@@ -11,13 +11,13 @@ Assuming you've gone through the required :doc:`deployment steps
 <fsconfig>` to setup at least one slave, initiating a call becomes
 very simple using the Switchy command line::
```
Comment (vodik) [inline docs/quickstart.rst]: And here

File: docs/quickstart.rst [hunk 8]
```diff
@@ -31,26 +31,26 @@ very simple using the Switchy command line::
 
 
 The Switchy `dial` sub-command takes several options and a list of minion IP addresses
```
Comment (vodik) [inline docs/quickstart.rst]: And here

File: docs/quickstart.rst [hunk 9]
```diff
@@ -70,14 +70,14 @@ What happens behind the scenes here is the following:
 
     * at the `with`, necessary internal Switchy components are instantiated in memory
```
Comment (vodik) [inline docs/quickstart.rst]: And here

File: docs/quickstart.rst [hunk 10]
```diff
@@ -70,14 +70,14 @@ What happens behind the scenes here is the following:
...
+    * the background :py:class:`~switchio.models.Job` returned by that command is handled
       to completion **synchronously** wherein the call blocks until the originating session has
       reached the connected state.
-    * the corresponding origininating :py:class:`~switchy.models.Session` is returned along with
-      a reference to a :py:meth:`switchy.handlers.EventListener.waitfor` blocker method.
-    * the call is kept up for 1 second and then :py:meth:`hungup <switchy.models.Session.hangup>`.
+    * the corresponding origininating :py:class:`~switchio.models.Session` is returned along with
+      a reference to a :py:meth:`switchio.handlers.EventListener.waitfor` blocker method.
+    * the call is kept up for 1 second and then :py:meth:`hungup <switchio.models.Session.hangup>`.
     * internal Switchy components are disconnected from the *FreeSWITCH* process at the close of the
```
Comment (vodik) [inline docs/quickstart.rst]: And here

File: docs/quickstart.rst [hunk 11]
```diff
@@ -112,9 +112,9 @@ If you have ``pytest`` installed you can run this test like so::
 
 Implementation details
 **********************
-The implementation of :py:func:`~switchy.sync.sync_caller` is shown
+The implementation of :py:func:`~switchio.sync.sync_caller` is shown
 below and can be referenced alongside the :doc:`usage` to gain a better
 understanding of the inner workings of Switchy's api:
```
Comment (vodik) [inline docs/quickstart.rst]: And here

File: docs/usage.rst [hunk 12]
```diff
@@ -20,21 +20,21 @@ Primary Components
 Currently there are 3 main objects in Switchy for driving
```
Comment (vodik) [inline docs/usage.rst]: And here

File: docs/usage.rst [hunk 13]
```diff
@@ -20,21 +20,21 @@ Primary Components
...
-:py:class:`~switchy.handlers.EventListener` - the type that contains the core
+:py:class:`~switchio.handlers.EventListener` - the type that contains the core
 event processing loop and logic
     - Primarily concerned with observing and tracking the state of
       a single *FreeSWITCH* process
     - Normally a one-to-one pairing of listeners to slave processes/servers
       is recommended to ensure deterministic control.
-    - Contains a :py:class:`~switchy.connection.Connection` used mostly for receiving
+    - Contains a :py:class:`~switchio.connection.Connection` used mostly for receiving
       events only transmitting ESL commands when dictated by :doc:`Switchy apps <apps>`
```
Comment (vodik) [inline docs/usage.rst]: Okay, at this point, I think its either forgot to grep for 'Switchy' with a capital S, or you don't care about what I'm pointing out.

Either way, I don't think I need to keep pointing every little issue out anymore.

**Comment by tgoodlet** [comment]
@vodik thanks for catching yeah it was an oversight.

**Pull request merged**